numpy/core/tests/test_numeric.py

```diff
@@ -48,6 +59,23 @@ def test_reshape_from_zero(self):
         assert_array_equal(Ar, np.zeros((2, 1), Ar.dtype))
         assert_equal(A.dtype, Ar.dtype)
 
+    def test_negative_resize(self):
+        A = np.arange(0, 10, dtype=np.float32)
+        new_shape = (-10, -1)
+        with pytest.raises(ValueError, match=r"negative"):
+            np.resize(A, new_shape=new_shape)
+
+    def test_subclass(self):
+        class MyArray(np.ndarray):
+            __array_priority__ = 1.
```

Comment:
What happens without this member?

Reply:
concatenate seems to lose the subclass info. That could actually be a bug, since all inputs are `MyArray`.

Reply:
Doesn't `concatenate` always lose subclass info?

Reply:
Hmm, no, the test would fail if it did :).